store/subscription.go

```diff
@@ -213,63 +213,68 @@ func (s *subscription) insert(ctx context.Context, events []*model.Transfer) (er
 	}
 
 	// Calculate tx fee
-	fees := make(map[gethCommon.Address]*big.Int)
+	fees := make(map[string]*big.Int)
```

Comment:
Suggest to use `common.Address` as key.

Reply:
Q: i didn't understand why we want to convert a 32 byte tx hash to 20 byte address?

Reply:
Ha, it's a bug and should convert to `common.Hash`.

Reply:
ha. thanks! i was confused. fixed.